TypeCobol.Test/Parser/Programs/TypeCobol/ProcedureCall.rdzPGM.txt

```diff
@@ -7,9 +7,9 @@ Line 37[21,26] <27, Error, Syntax> - Syntax error : Parameter with name 'format'
 Line 38[21,24] <27, Error, Syntax> - Syntax error : Parameter with name 'var1' declared multiple times OffendingSymbol=[21,24:var1]<UserDefinedWord>
 Line 39[21,24] <27, Error, Syntax> - Syntax error : Parameter with name 'var1' declared multiple times OffendingSymbol=[21,24:var1]<UserDefinedWord>
 Line 41[21,26] <27, Error, Syntax> - Syntax error : Parameter with name 'format' declared multiple times OffendingSymbol=[21,26:format]<UserDefinedWord>
-Line 111[8,55] <27, Error, Syntax> - Syntax error : A function "proc2" with the same profile already exists in namespace "ProcedureCall".
-Line 115[8,55] <27, Error, Syntax> - Syntax error : A function "proc2" with the same profile already exists in namespace "ProcedureCall".
-Line 119[8,70] <27, Error, Syntax> - Syntax error : A function "proc2" with the same profile already exists in namespace "ProcedureCall".
+Line 111[8,14] <27, Error, Syntax> - Syntax error : A function "proc2" with the same profile already exists in namespace "ProcedureCall". OffendingSymbol=[8,14:declare]<DECLARE>
```

Comment:
Should not it be a Semantic error here ?

Reply:
done.
https://github.com/TypeCobolTeam/TypeCobol/pull/1669/commits/4b5e1d0af58323248efbd75657d4b554eb7db081